tests/jballerina-semtype-port-test/src/test/java/io/ballerina/semtype/port/test/SemTypeAssertionTransformer.java

```diff
@@ -118,16 +124,38 @@ private SemType listProj(Context context, SemType t, SemType m) {
     }
 
     private String[] splitAssertion(String str) {
+        if (ignoredCommentSet().contains(str)) {
+            return null;
+        }
         String[] parts = str.split(" ");
+
+        if (parts[1].equals("-@type")){
+            // TODO: remove this check once diff operator is supported
```

Comment:
At the moment `!` operator is not supported in the tests. Those lines are ignored with `-@type` instead of disabling the entire bal file. e.g. `xml-complex-rw-tv.bal` and `not1-tv.bal`